deploy-service/teletraanservice/src/main/java/com/pinterest/teletraan/exception/GenericExceptionMapper.java

```diff
@@ -72,10 +72,12 @@ public Response toResponse(Throwable t) {
                 return Response.serverError().entity(sb.toString()).build();
             } else {
                 sb.append(errorMessage);
+                /*
```

Comment:
Please just remove commented out code

Reply:
Also, there is another place at line 49 in the file dumping stack trace.

Reply:
yea, I will remove the comments.
but are you really sure you want me to remove the first branch around line 49?
That is another branch which I have no idea what it is...

Reply:
That's adjacent area which requires similar improvement but If you are not sure, let's have a followup on that. we can resolve this PR.